lib/themes/dosomething/paraneue_dosomething/js/components/ReactionsList.js

```diff
@@ -101,7 +101,8 @@ class ReactionsList extends React.Component {
 
     // Delete specified reaction record.
     if (reactionId) {
-      sendEvent('Onboarding', 'Un-like');
+      sendEvent('Onboarding', 'Un-like'); // Soon to be removed.
+      sendEvent('onboarding-v2', 'unlike', this.constructor.name);
```

Comment:
Will `this.constructor.name` still provide a useful value if the code is minified?